src/server/main_service.cc

```diff
@@ -902,17 +902,21 @@ OpStatus CheckKeysDeclared(const ConnectionState::ScriptInfo& eval_info, const C
   if (!key_index_res)
     return key_index_res.status();
 
+  // TODO: Switch to transaction internal locked keys once single hop multi transactions are merged
+  // const auto& locked_keys = trans->GetMultiKeys();
+  const auto& locked_keys = eval_info.keys;
+
```

Comment:
I can't use this because of our single hop multi transaction "fix". Once proper single hop multi transactions are merged, we can use it 😭